internal/github/wrapper.go

```diff
@@ -0,0 +1,25 @@
+package github
+
+import (
+	"context"
+
+	"github.com/google/go-github/v32/github"
+)
+
+type GithubClient interface {
```

Comment:
As this is the `github` package, is suggested to not duplicate in in structs or functions names, like `github.GithubClient`.

Remove `Github` from these names:
```
GithubClient -> Client
GithubWrapper -> Wrapper
NewGithubWrapper -> NewWrapper
```